build_tools/github_actions/configure_ci.py

```diff
@@ -256,14 +247,10 @@ def main():
   else:
     output["should-run"] = "false"
   output[RUNNER_ENV_KEY] = get_runner_env(trailers)
-  ci_stage = get_ci_stage(event_name)
-  output["ci-stage"] = ci_stage
-  output["runner-group"] = ci_stage
-  write_caches = "0"
-  if ci_stage == "postsubmit":
-    write_caches = "1"
-  output["write-caches"] = write_caches
-  output["benchmark-presets"] = get_benchmark_presets(ci_stage, trailers)
+  on_fork = run_on_fork(event_name)
+  output["runner-group"] = "presubmit" if on_fork else "postsubmit"
+  output["write-caches"] = "0" if on_fork else "1"
```

Comment:
This condition seems weird (related to the helper function). Do we want to write to caches from all "trusted" jobs, or only from postsubmit jobs?

Reply:
All trusted jobs was the idea. And same with the runners. These are really about trust, not whether it's a PR. Although given the current fork detection logic it's the same. I guess it might also be confusing to see your PR running on "postsubmit" runners. Would also mess with our quota a bit... Hmmm... I guess I could change this to "is a PR" and then would this all make sense to you?

Reply:
Yep, that makes sense to me now. I think generally I'm looking for some of the policy/security decisions to be documented more clearly: "we use this runner group when x condition is met, because y", "we write to caches when x is true, otherwise we just read from caches". This PR shows just how many configurations of events we have to deal with.